lib/dfe/analytics/transaction_changes.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
```

Comment:
I'm not sure I follow this entirely as it's a bit esoteric. Is this an existing solution to the problem ?

Reply:
Yes its mainly from https://github.com/dylanahsmith/ar_transaction_changes/blob/main/README.md

Reply:
It basically adds the transaction_changed_attributes to a model